Traceback (most recent call last):
  File "train_res50_fpn.py", line 246, in <module>
    main(args)
  File "train_res50_fpn.py", line 161, in main
    coco_info = utils.evaluate(model, val_data_set_loader, device=device)
  File "/root/miniconda3/envs/frcnn/lib/python3.8/site-packages/torch/autograd/grad_mode.py", line 27, in decorate_context
    return func(*args, **kwargs)
  File "/root/code/Multi-labelClothingDetection/code/pytorch_object_detection/faster_rcnn/train_utils/train_eval_utils.py", line 76, in evaluate
    coco = get_coco_api_from_dataset(data_loader.dataset)
  File "/root/code/Multi-labelClothingDetection/code/pytorch_object_detection/faster_rcnn/train_utils/coco_utils.py", line 54, in get_coco_api_from_dataset
    return convert_to_coco_api(dataset)
  File "/root/code/Multi-labelClothingDetection/code/pytorch_object_detection/faster_rcnn/train_utils/coco_utils.py", line 15, in convert_to_coco_api
    hw, targets = ds.coco_index(img_idx)
  File "/root/code/Multi-labelClothingDetection/code/pytorch_object_detection/faster_rcnn/my_dataset.py", line 191, in coco_index
    xml = etree.fromstring(xml_str)
  File "src/lxml/etree.pyx", line 3257, in lxml.etree.fromstring
  File "src/lxml/parser.pxi", line 1911, in lxml.etree._parseMemoryDocument
ValueError: Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration.
